modules/enforcers/templates/values.yaml

```diff
@@ -174,9 +174,11 @@ webhooks:
   failurePolicy: Ignore
   validatingWebhook:
     name: "kube-enforcer-admission-hook-config"
+    timeout: 2
     annotations: {}
   mutatingWebhook:
     name: "kube-enforcer-me-injection-hook-config"
+    timeout: 2
```

Comment:
@uchinda-sph their chart default value is not taking place ?

Reply:
Nope it's not
![Screenshot 2023-04-17 at 5 42 07 PM](https://user-images.githubusercontent.com/89894943/232447547-9527c6d9-d747-485a-95c7-e8a57d4fd94c.png)